### PR TITLE
Change metricValue & maxMetricValue to float 

### LINKED ISSUE #388: Tool Use Profile: Update AggregateMeasure metricValue & maxMetricValue to float
currently they are integer but the fixtures use floats

### PR DESCRIPTION
**Pull request by andrew-gardener**
Also update json-ld to match fixtures

Closes #388

### CODE REVIEW

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html [hunk 1]
```diff
@@ -307,16 +307,16 @@ <h3>AggregateMeasure</h3>
                 <td>Required</td>
             </tr>
             <td>metricValue</td>
-            <td>integer</td>
-            <td>An integer value representing the total curricular progress the actor has made in the context of the
+            <td>float</td>
+            <td>An float value representing the total curricular progress the actor has made in the context of the
```
Comment (amiller-ims) [inline profiles/tooluse/caliper-profile-tool_use-v1p1.html]: Should be "a float"

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html [hunk 2]
```diff
@@ -307,16 +307,16 @@ <h3>AggregateMeasure</h3>
...
                 application.
             </td>
             <td>Required</td>
             </tr>
             <tr>
                 <td>maxMetricValue</td>
-                <td>integer</td>
-                <td>An integer value representing the total possible curricular progress the actor can make in the context of
+                <td>float</td>
+                <td>An float value representing the total possible curricular progress the actor can make in the context of
```
Comment (amiller-ims) [inline profiles/tooluse/caliper-profile-tool_use-v1p1.html]: "A float"

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html [hunk 3]
```diff
@@ -307,16 +307,16 @@ <h3>AggregateMeasure</h3>
                 <td>Required</td>
             </tr>
             <td>metricValue</td>
-            <td>integer</td>
-            <td>An integer value representing the total curricular progress the actor has made in the context of the
+            <td>float</td>
```
Comment (arwhyte) [inline profiles/tooluse/caliper-profile-tool_use-v1p1.html]: @andrew-gardener -- The value type Caliper uses for expressing non-integer numbers is `decimal` not `float`. See the ontology and/JSON-LD context for examples.  

JSON-LD context:
```
"maxMetricValue": {"@id": "caliper:maxMetricValue", "@type": "xsd:decimal"},
...
"maxValue": {"@id": "caliper:maxValue", "type": "xsd:decimal"},
```

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html (hunk 1 above)
Comment (andrew-gardener) [reply profiles/tooluse/caliper-profile-tool_use-v1p1.html]: fixed

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html (hunk 2 above)
Comment (andrew-gardener) [reply profiles/tooluse/caliper-profile-tool_use-v1p1.html]: fixed

File: profiles/tooluse/caliper-profile-tool_use-v1p1.html (hunk 3 above)
Comment (andrew-gardener) [reply profiles/tooluse/caliper-profile-tool_use-v1p1.html]: fixed

**Pull request merged**